src/ScottPlot/Palettes/DarkLight.cs

```diff
@@ -0,0 +1,15 @@
+﻿/* This a qualitative 8-color palette generated using:
+ * https://colorbrewer2.org
+ * which is the lighter-color version of the 'Dark' palette.
+ */
+namespace ScottPlot.Drawing.Colorsets
+{
+    class DarkLight : HexColorset, IPalette
+    {
+        public override string[] hexColors => new string[]
+        {
+            "#66c2a5", "#fc8d62", "#8da0cb", "#e78ac3", "#a6d854",
```

Comment:
I'm not sure if these are just x% lighter than `Dark`, but if so you might be able to do that programmatically? I'm fine with having constants given that every other palette is done like this, and it's not like the pallettes are likely to change. Depends if you think there's a benefit.

Reply:
> might be able to do that programmatically?

True, but I think there's advantage to explicitly writing-out the colors. It makes it easier to use these colors elsewhere, and I'm hesitant to start encouraging coupling between palettes 🤔